charts/ai-software-templates/pipeline-install/templates/app-config.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    helm.sh/hook: post-install,post-upgrade
+    helm.sh/hook-delete-policy: hook-succeeded
+  labels:
+    app.kubernetes.io/instance: {{ .Release.Name }}-app-config
+    app.kubernetes.io/name:  {{ .Release.Name }}-app-config
+    app.kubernetes.io/part-of: {{ .Release.Name }}
+  name: {{ .Release.Name }}-app-config
+  namespace: {{ .Release.Namespace }}  
+data:
+  APP_NAME: "{{ .Release.Name }}"
+  APP_NAMESPACE: "{{ .Release.Namespace }}"
+  GITHUB_APP_APP_ID: "{{ .Values.tekton.gitAppId }}"
+  GITHUB_APP_WEBHOOK_SECRET: "{{ .Values.tekton.gitAppWebhookSecret }}"
```

Comment:
Does it make sense to move this one too to the `app-secret.yaml`?

Reply:
the webhook secret is pretty benign ... it is just a cross verification vs. used for actual authentication.
like the github token or private key.

I'll think about it